src/main/java/org/takes/rs/RsWithCookie.java

```diff
@@ -74,7 +80,13 @@ public RsWithCookie(final Response res, final String name,
         final String value, final String... attrs) {
         super(
             new RsWithHeader(
-                res, "Set-Cookie", RsWithCookie.make(name, value, attrs)
+                res,
+                "Set-Cookie",
+                RsWithCookie.make(
+                    RsWithCookie.checkName(name),
+                    RsWithCookie.checkValue(value),
+                    attrs
+            )
```

Comment:
@pinaf this is a bug in the checkstyle.
it reports  `method call rparen at indentation level 16 not at correct indentation, 12 (IndentationCheck)`
should I make good indentation and disable IndentationCheck here ?